### ISSUE TITLE
Computer is not a constructor

### ISSUE THREAD

**Issue by corper** [bitcoin-computer/bitcoin-computer-lib#3]
```
import Computer from 'bitcoin-computer';
const computer = new Computer({
    seed: ',
    chain: 'BSV', // BSV or BCH
    network: 'testnet' // testnet or livenet
});
```
It shows error below when I run the codes above:
`TypeError: Computer is not a constructor`

I change the code to `const computer = new Computer.default(...)` to fix it.

**Comment by amoghaddassi** [comment]
Thank you!

**Comment by stonekase** [comment]
> Thank you! Saved me a major headache

Didn't work for me

**Comment by ClemensLey** [comment]
Hi @corper, @amoghaddassi, @stonekase. Thanks for your input and apologies for the late reply. This issue is fixed by version 0.5.x-beta. The computer object is now a named export and can be imported like this:

```
import { Computer } from 'bitcoin-computer-lib'
```

Have a look at the revised readme file for more info.
